src/esp/gfx/replay/Keyframe.h

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
I'm introducing this replay subfolder because I have a lot of files incoming for render-replay.

Reply:
The render-replay feature is logically a subset of `gfx`. We will probably have a Sim Replay feature at some point; it would *not* use this Keyframe class. My placement of render-replay files under `gfx` will help distinguish those files from Sim Replay files.